CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ option( TRACCC_BUILD_CUDA "Build the CUDA sources included in traccc"
    ${TRACCC_BUILD_CUDA_DEFAULT} )
 option( TRACCC_BUILD_SYCL "Build the SYCL sources included in traccc" FALSE )
 option( TRACCC_BUILD_TESTING "Build the (unit) tests of traccc" TRUE )
+option( TRACCC_BUILD_EXAMPLES "Build the examples of traccc" FALSE )
```

Comment:
I think the default should be `TRUE` for this option. Since this is our "top project" for the time being, I like to be able to build it with a minimal set of options specified on the command line.

Reply:
Ok it does work